python/tvm/relay/backend/contrib/ethosu/tir/transform.py

```diff
@@ -58,4 +58,5 @@ def get_copy_params(stmt, producers, consumers):
         ),
         write_store.buffer_var,
         None,
+        True,
```

Comment:
The docstring for the last two return values is missing.

Reply:
Done